src/test/resources/features/theme 1/9 - aedm and aed.feature

```diff
@@ -111,3 +111,42 @@ Feature: 09 - AEDM and AED
     And I click the "Test quality information" link
     When I click the TQI link for tester {testerUsername}
     Then The page contains "{testerName}"
+
```

Comment:
Can you create a new file under Theme 4 called "4 - Transaction History Check as AEDM" with the tag @cpms for these scenarios